src/handbook/peopleops/compensation.md

```diff
@@ -16,11 +16,31 @@ on compensation expectation of the candidate as well as the market rates. Salary
 is based on your location you live in and the company context. FlowForge pays in
 your local currency unless explicitly discussed otherwise.
 
+## Transition to Single-Point Increase Cycle and Inflation Correction
+
+We are making some changes to how we handle salary increases at FlowForge. We're moving away from the current two-point increase cycle and adopting a single-point increase cycle. This will make the process more streamlined and predictable for everyone. Along with this change, we will also be incorporating inflation correction for applicable countries. 
+
+With the single-point increase cycle, salary adjustments will happen once a year, aligning with our annual budgeting and planning process. This will provide a clear timeline for adjustments.
+
+In applicable countries, we will be adjusting salaries to account for inflation rates. This ensures that employees' purchasing power is maintained. The adjustment percentage will be based on reliable inflation data from trusted sources. 
```

Comment:
may want to make this a little less commital given the indent at the end of 2022. To me, this reads as - you get a pay increase inline with whatever inflation is?